scripts/run_failpoint_tests.py

```
@@ -0,0 +1,115 @@
+import subprocess
+import sys
+import os
+from collections import defaultdict
+
+
+def main() -> int:
+    repo_root = pathlib.Path(__file__).resolve().parent.parent
+    env = os.environ.copy()
+    failpoint_ctl = (
+        pathlib.Path(
+            subprocess.check_output(["go", "env", "GOPATH"], text=True).strip()
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)